### PR TITLE
docs: update network.md and errors.md

### PR DESCRIPTION
**Pull request by hopeyen**
Update 
- testnet's network parameter and indexer component recommended versions
- Env var for `--epoch-subgraph-endpoint` set for subgraph `graphprotocol/goerli-epoch-block-oracle`
- error.md for IE070

### CODE REVIEW

File: docs/networks.md [hunk 1]
```diff
@@ -47,18 +47,19 @@ indexer. The only requirement is a minimum stake of 100k GRT.
...
+| `INDEXER_AGENT_ETHEREUM_NETWORK`            | `--ethereum-network`            | `mainnet`                                                                         |
+| `INDEXER_AGENT_INDEXER_ADDRESS`             | `--indexer-address`             | Ethereum address of mainnet indexer                                               |
+| `INDEXER_AGENT_INDEXER_GEO_COORDINATES`     | `--indexer-geo-coordinates`     | Geo coordinates of mainnet indexer infrastructure                                 |
+| `INDEXER_AGENT_MNEMONIC`                    | `--mnemonic`                    | Ethereum mnemonic for mainnet operator                                            |
+| `INDEXER_AGENT_NETWORK_SUBGRAPH_DEPLOYMENT` | `--network-subgraph-deployment` | `QmV614UpBCpuusv5MsismmPYu4KqLtdeNMKpiNrX56kw6u`                                  |
+| `INDEXER_AGENT_NETWORK_SUBGRAPH_ENDPOINT`   | `--network-subgraph-endpoint`   | `https://gateway.thegraph.com/network`                                            |
+| `INDEXER_AGENT_EPOCH_SUBGRAPH_ENDPOINT`     | `--epoch-subgraph-endpoint`     | `https://api.thegraph.com/subgraphs/name/graphprotocol/goerli-epoch-block-oracle` |
```
Comment (fordN) [inline docs/networks.md:59]: This should not be set on Mainnet yet, just on Testnet/Goerli, so this line should be moved to ~190.

File: docs/errors.md [hunk 2]
```diff
@@ -872,7 +872,7 @@ Failed to query BlockHashFromNumber from graph node
 
 **Solution**
 
-Graph-node could not find the block hash given network and block number, check if graph-node has access to a network client that has synced to the required block. 
+Graph-node could not find the block hash given network and block number, check if graph-node has access to a network client that has synced to the required block, and that the network is configured in the default index-node.
```
Comment (fordN) [inline docs/errors.md:875]: What is meant by "default index-node" here?

File: docs/networks.md (hunk 1 above)
Comment (hopeyen) [reply docs/networks.md:59]: removed! (I imagine this endpoint also works for mainnet in theory?

File: docs/errors.md (hunk 2 above)
Comment (hopeyen) [reply docs/errors.md:875]: updating this to be `the same index-node`? 
It seems that when indexers query blockHashFromNumber, the network had to be on the same index node, but I guess there's no `default`

Comment (fordN) [reply docs/errors.md]: Sorry this still isn't crystal clear to me. Same index node as what?

Comment (fordN) [reply docs/errors.md]: Do we mean, ensure that index node is configured with an RPC endpoint from the given network?

File: docs/networks.md (hunk 1 above)
Comment (fordN) [reply docs/networks.md]: Ok, but we need to add it for the testnet configs!

**Review by fordN — APPROVED** [review]
👍

**Pull request merged**